model/internal/otlp_wrappers_test.go

```diff
@@ -114,3 +117,109 @@ func TestDeprecatedStatusCode(t *testing.T) {
 		})
 	}
 }
+
+func TestDeprecatedIntHistogram(t *testing.T) {
```

Comment:
I think is better to have "input proto" expected "output proto" the whole histogram, not individual parts of the proto. This is nit, but I think fits better here.

Reply:
Sorry, now that I see, I think we can limit to `inputMetrics: []*otlpmetrics.Metric`  as the input and we can constructor the request and extract from response the `outputMetrics []*otlpmetrics.Metric` since the rest is not necessarily important. Helps a bit with indentation.

Reply:
No problem, i agree that limiting this to Metric makes indentation easier to read.